simapp/app_config.go

```diff
@@ -48,7 +48,6 @@ import (
 	vestingtypes "github.com/cosmos/cosmos-sdk/x/auth/vesting/types"
 	"github.com/cosmos/cosmos-sdk/x/authz"
 	_ "github.com/cosmos/cosmos-sdk/x/authz/module" // import for side-effects
-	_ "github.com/cosmos/cosmos-sdk/x/bank"         // import for side-effects
```

Comment:
Can't this be used? I think it's enough to remove inactive address in bankplus module and return empty output on ProvideModule.

Reply:
`_ "github.com/cosmos/cosmos-sdk/x/bank"` can be added. 
Bankplus already imports the original bank package, so it has no effect to add the import or not.

Is it okay to return empty output in bankplus's provider??
I'm not sure, but it needs to provide bankplus'  appmodule to register migration logic(removing inactive addresses).

Reply:
Of cause, bankplus's appmodule is needed but I think it doesn't need to provide bank keeper.